lametro/people.py

```diff
@@ -24,7 +24,7 @@
                 'Don Knabe' : 'Los Angeles County Board Supervisor, District 4',
                 'Michael Antonovich' : 'Los Angeles County Board Supervisor, District 5'}
 
-NONVOTING_POSTS = {'Carrie Bowen' : 'Appointee of Governor of California'}
+NONVOTING_POSTS = {'Shirley Choate' : 'Acting Caltrans District 7 Director, Appointee of Governor of California'}
```

Comment:
Why are we replacing Carrie Bower in this dict, shouldn't we just add Shirley Coate?

Reply:
That's right.